ktor-server/ktor-server-plugins/ktor-server-caching-headers/jvmAndNix/src/io/ktor/server/plugins/cachingheaders/CachingHeaders.kt

```diff
@@ -79,3 +80,13 @@ public val CachingHeaders: RouteScopedPlugin<CachingHeadersConfig> = createRoute
         }
     }
 }
+
+/**
+ * Gets or sets the [CacheControl] for this call.
+ */
+public var ApplicationCall.caching: CachingOptions?
```

Comment:
should we name it `cachingOptions`?

Reply:
I named it the same way as in `OutgoingContent` for consistency.